### PR TITLE
Merge master with federation

### PR DESCRIPTION
**Pull request by sorin-stefan-iovita-sap**
### Description:
Pull request that merges latest changes from master into the federation branch.

### CODE REVIEW

File: services/distribution/src/main/resources/application.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ services:
     app-features:
       - label: reserved
         value: ${reserved:1}
-    supported-countries: ${SUPPORTED_COUNTRIES}
+    supported-countries: ${SUPPORTED_COUNTRIES:DE}
```
Comment (hilmarf) [inline services/distribution/src/main/resources/application.yaml:88]: please remove default value

File: services/submission/pom.xml [hunk 2]
```diff
@@ -67,6 +67,12 @@
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java-util</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
+      <version>3.12.4</version>
+      <scope>test</scope>
```
Comment (hilmarf) [inline services/submission/pom.xml:74]: please remove <version> + <scope> as it is already defined in the managed dependency section of the root-pom

File: services/distribution/src/test/java/app/coronawarn/server/services/distribution/runner/TestDataGenerationTest.java [hunk 3]
```diff
@@ -35,6 +35,7 @@
 import app.coronawarn.server.services.distribution.assembly.structure.util.TimeUtils;
 import app.coronawarn.server.services.distribution.config.DistributionServiceConfig;
 import app.coronawarn.server.services.distribution.config.DistributionServiceConfig.TestData;
+import org.junit.After;
```
Comment (pithumke) [inline services/distribution/src/test/java/app/coronawarn/server/services/distribution/runner/TestDataGenerationTest.java:38]: Unused import?

**Review by hilmarf — CHANGES_REQUESTED** [review]
I can judge only on the `pom.xml` and `*.yaml` files... don't know about the tests etc.

File: services/distribution/src/main/resources/application.yaml (hunk 1 above)
Comment (sorin-stefan-iovita-sap) [reply services/distribution/src/main/resources/application.yaml]: done

File: services/submission/pom.xml (hunk 2 above)
Comment (sorin-stefan-iovita-sap) [reply services/submission/pom.xml]: done

File: services/distribution/src/test/java/app/coronawarn/server/services/distribution/runner/TestDataGenerationTest.java (hunk 3 above)
Comment (sorin-stefan-iovita-sap) [reply services/distribution/src/test/java/app/coronawarn/server/services/distribution/runner/TestDataGenerationTest.java]: done

**Review by hilmarf — APPROVED** [review]

**Pull request merged**